website/docs/docs/deploy/dbt-cloud-job.md

```diff
@@ -37,6 +37,19 @@ Refer to [Scheduler](/docs/deploy/job-scheduler) for more info on how the schedu
 
 <Lightbox src="/img/docs/dbt-cloud/deployment/deploy-scheduler.jpg" width="100%" title="An overview of a dbt Cloud job run"/>
 
+### Deployment monitor
+
+Use the Deployment monitor to gain greater visibility into your deployment health and quickly detect any issues with the improved filtering and sorting capabilities. You can now easily curate the view based on environment, prioritizing the ones that matter most, such as Production. With pre-filtered lists for in progress and in queue activities, it's quicker and easier to track the relevant activity and stay on top of your deployment status.
```

Comment:
suggs: 

- first sentence reads like it's referring to the actual page/view in the UI which is shown in the screenshot below. sugg matching capitalization and bolding it since it's a UI label
- conciseness 
- https://www.merriam-webster.com/dictionary/prefilter -- merriam webster is my personal source of truth for American English spellings

```suggestion
Use the **Deployment Monitor** view for greater visibility into your deployment health and to quickly detect any issues with the filtering and sorting capabilities. You can curate the view based on the environment, such as Production. With prefiltered lists for **In Progress** and **In Queue activities, it's quicker and easier to track the relevant activity and your deployment status.
```